rnndb/bus/pci.xml

```diff
@@ -489,6 +489,19 @@
 			<value value="2" name="2_5GT"/>
 		</bitfield>
 	</reg32>
+	<reg32 offset="0x1c0" name="UNK1">
+		<bitfield low="16" high="17" name="LNK_CAP_SPEED">
+			<value value="3" name="8_0GT"/>
+			<value value="2" name="5_0GT"/>
+			<value value="1" name="2_5GT"/>
```

Comment:
okay, I also added a reg for pcie_v2 support, though I am not 100% sure, but its only set for cards with SYSTEM_SPEED=5.0 or 8.0, so it might be the case